### DISCUSSION TITLE
Software for icon design

### DISCUSSION THREAD

**Discussion by map-per** [organicmaps/organicmaps#5917] [Q&A] [answered]
What software can you recomend / do you use for designing map icons? Editing .svg code directly is rather cumbersome.

**Comment by hb0nd** [comment, marked as answer]
https://inkscape.org/ is the best option :)

**Reply by map-per** [reply]
Thanks!

**Comment by RicoElectrico** [comment]
If you don't mind one-time purchase, another option is Affinity Designer. It has Pixel View, which can view icons in rasterized form as you're drawing them. They seem to cater to the UI designer crowd with a few of their features.
